feature/foryou/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -444,6 +473,38 @@ fun TopicIcon(
     )
 }
 
+@Composable
+private fun ShowDialog(
+    onDismissRequest: ()-> Unit,
+    askNotificationPermission: () -> Unit
+) {
+    AlertDialog(
+        onDismissRequest = onDismissRequest,
+        title = {
+            Text(text = "Grant Notification Permission")
+        },
+        text = {
+            Text("Would you like to be informed when new content is published on these topics, content is typically published every few days?")
```

Comment:
String resources should be placed in `strings.xml`

Reply:
@arvindri2005 Please do not resolve comments before they have actually been resolved.

Reply:
I have fixed this in the latest commit cfa1d51